SPDYUnitTests/SPDYURLRequestTest.m

```diff
@@ -314,6 +318,41 @@ - (void)testSPDYProperties
     STAssertEquals(immutableCopy.SPDYBypass, (BOOL)TRUE, nil);
     STAssertEquals(immutableCopy.SPDYBodyStream, stream, nil);
     STAssertEquals(immutableCopy.SPDYBodyFile, @"Bodyfile.json", nil);
+    STAssertEquals(immutableCopy.SPDYURLSession, urlSession, nil);
+}
+
+- (void)testRequestCopyDoesRetainProperties
+{
+    NSURL *url = [[NSURL alloc] initWithString:@"http://example.com/test/path"];
+    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:url];
+    NSInputStream *stream = [[NSInputStream alloc] initWithData:[NSData new]];
+    NSURLSession *urlSession = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
+
+    request.SPDYBodyStream = stream;
+    request.SPDYBodyFile = @"Bodyfile.json";
+    request.SPDYURLSession = urlSession;
+
+    NSInputStream __weak *weakStream = stream;
+    NSURLSession __weak *weakURLSession = urlSession;
+    NSMutableURLRequest __weak *weakRequest = request;
+    NSURLRequest *immutableCopy = [request copy];
+
+    @autoreleasepool {
```

Comment:
It's a bit over-exuberant, but the point of the test is: create a request with all reference config properties set, copy it, release the original request, then verify all the references in the copy are still valid and still point to the original objects. It's most relevant to the new SPDYURLSession, since it uses an indirect associated object technique -- want to make sure it survives the request copy.